packages/connection-form/src/hooks/use-connection-color.ts

```diff
@@ -161,22 +160,6 @@ export function useConnectionColor(): {
     [isDarkMode]
   );
 
-  const colorCodeToHex = useCallback(
-    (colorCode: string | undefined): string | undefined => {
-      if (!colorCode) {
-        return;
-      }
-
-      const migratedColor = legacyColorsToColorCode(colorCode);
```

Comment:
We only have one other use of `legacyColorsToColorCode`, and it's in the color picker. Maybe we can remove `legacyColorsToColorCode` and `LEGACY_COLORS_TO_COLOR_CODE_MAP` now? Someone would be using a color from ~3 years ago, and we'd only slightly be helping them select a color inside of the color picker UI.

Reply:
that must be part of the legacy bits. I'll try to remove it + the parts that use it, hopefully it's not too deep of a rabbit hole 🤞